code/game/objects/items.dm

```diff
@@ -654,7 +654,7 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 		zoom = !zoom
 		if(user.interactee)
 			user.unset_interaction()
-		else
+		else if(!istype(src, /obj/item/attachable/scope))
```

Comment:
Make this `istype(src, /obj/item/device/binoculars/tactical)` or add the lines to a child version of this proc, since it's the only zoom item that makes use of handle_click().

Reply:
This is more of a future proofing, I don't expect someone making a new zoomable item with special interaction adding it here.

Reply:
That's also true.